manager/src/main/resources/db/migration/0.1/V0.1.27__add_subscription_id_for_bridges.sql

```diff
@@ -0,0 +1,2 @@
+alter table BRIDGE
+    add column "subscription_id" varchar(255);
```

Comment:
@r00ta For consistency, I think we should also:

- Generate and insert a random uuid for already existing bridges, I guess using https://www.postgresql.org/docs/13/functions-uuid.html
- Set the column to `not null`.

When we extend this work to integrate with AMS, we'll have to consider some sort of reconciliation process to map existing bridges to a new `subscription_id` generated by AMS, but at least our data structures are consistent from the start.

Reply:
done